boards/BMS/Src/App/App_Accumulator.c

```diff
@@ -79,8 +95,19 @@ void App_Accumulator_RunOnTick100Hz(
             // Read cell voltages
             accumulator->read_cell_voltages();
 
-            // TODO: Add broadcast BMS CAN messages when fixed pt. CAN PR has
-            // been merged in
+            // Write to configuration registers
+            accumulator->write_to_cfg_reg();
+
+            // If current state machine state is the charge state, disable
```

Comment:
weird that comment has the opposite logic than the line below it, and also just restates the code